app/controllers/register/establishers/partnership/partner/ConfirmDeletePartnerController.scala

```diff
@@ -42,23 +44,30 @@ class ConfirmDeletePartnerController @Inject()(
                                                 authenticate: AuthAction,
                                                 getData: DataRetrievalAction,
                                                 requireData: DataRequiredAction,
-                                                sectionComplete: SectionComplete
+                                                sectionComplete: SectionComplete,
+                                                formProvider : ConfirmDeletePartnerFormProvider
                                               ) (implicit val ec: ExecutionContext) extends FrontendController with I18nSupport with Retrievals {
 
+  private val form: Form[Boolean] = formProvider()
+
   def onPageLoad(establisherIndex: Index, partnerIndex: Index): Action[AnyContent] = (authenticate andThen getData andThen requireData).async {
     implicit request =>
       (PartnershipDetailsId(establisherIndex) and PartnerDetailsId(establisherIndex, partnerIndex)).retrieve.right.map {
         case partnership ~ partner =>
           if (partner.isDeleted) {
             Future.successful(Redirect(routes.AlreadyDeletedController.onPageLoad(establisherIndex, partnerIndex)))
           } else {
+            val preparedForm = request.userAnswers.get(ConfirmDeletePartnerId(establisherIndex)) match {
```

Comment:
Since this is a confirm page, it should not be pre-populated with existing data

Reply:
fixed